src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -242,6 +242,9 @@ class CLISecurityDomainOperation(str, Enum):
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.', validator=validate_subnet)
 
+    with self.argument_context('keyvault network-rule add', min_api='2018-02-14') as c:
+        c.argument('ip_address', nargs='*', help='IPv4 address or CIDR range. Can supply a list: --ip-address ip1 [ip2]...', validator=validate_ip_address)
```

Comment:
```suggestion
        c.argument('ip_address', nargs='+', help='IPv4 address or CIDR range. Can supply a list: --ip-address ip1 [ip2]...', validator=validate_ip_address)
```
At least one address needed, so use `+` instead of `*`.